src/test/java/org/jabref/logic/integrity/CitationKeyCheckerTest.java

```diff
@@ -13,14 +15,37 @@
 public class CitationKeyCheckerTest {
 
     private final CitationKeyChecker checker = new CitationKeyChecker();
-    private final BibEntry entry = new BibEntry();
 
     @Test
     void bibTexAcceptsKeyFromAuthorAndYear() {
-        entry.setField(InternalField.KEY_FIELD, "Knuth2014");
-        entry.setField(StandardField.AUTHOR, "Knuth");
-        entry.setField(StandardField.YEAR, "2014");
+        BibEntry entry = new BibEntry().withField(InternalField.KEY_FIELD, "Knuth2014")
+                                       .withField(StandardField.AUTHOR, "Knuth")
+                                       .withField(StandardField.YEAR, "2014");
         assertEquals(Collections.emptyList(), checker.check(entry));
     }
 
+    @Test
+    void KeyFromAuthorAndTitle() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Thanks for the feedback! Changed it in commit [93ab8a4](https://github.com/JabRef/jabref/pull/7542/commits/93ab8a40d1b513b5f608b2fc60a0b9323012c2fa)

Reply:
Happy to help.